code/game/machinery/computer/store.dm

```diff
@@ -20,115 +83,62 @@
 	src.add_hiddenprint(user)
 	return attack_hand(user)
 
-/obj/machinery/computer/merch/attack_hand(mob/user as mob)
-	user.set_machine(src)
-	add_fingerprint(user)
+/obj/machinery/computer/merch/proc/get_display_name(var/datum/storeitem/storeitem)
+	return "[storeitem.name] ([!(storeitem.cost) ? "free" : "[storeitem.cost]$"])"
+
+/obj/machinery/computer/merch/attack_hand(var/mob/user)
+	. = ..()
+	if(.)
+		interface.hide(user)
+		return
+
+	interact(user)
+
+/obj/machinery/computer/merch/interact(mob/user)
+	if(stat & (BROKEN|NOPOWER))
+		return
+	if(user.stat || user.restrained() || !allowed(user))
+		return
+
+	var/delay = 0
+	delay += send_asset(user.client, "Chart.js")
```

Comment:
Just remove references to `Chart.JS`, it's the javascript library used for the power monitor's chart.